llama_index/query_engine/sub_question_query_engine.py

```diff
@@ -64,16 +65,20 @@ def from_defaults(
         query_engine_tools: Sequence[QueryEngineTool],
         question_gen: Optional[BaseQuestionGenerator] = None,
         response_synthesizer: Optional[ResponseSynthesizer] = None,
+        service_context: Optional[ServiceContext] = None,
         verbose: bool = True,
         use_async: bool = True,
     ) -> "SubQuestionQueryEngine":
         callback_manager = None
         if len(query_engine_tools) > 0:
             callback_manager = query_engine_tools[0].query_engine.callback_manager
```

Comment:
We could also get the callback manager from the service context now, although I'm not sure which callback_manager should take precedence 🤔

Reply:
Ya, I'm also confused about that. Will leave that decision for the future.